server/src/main/java/io/druid/server/coordinator/rules/IntervalLoadRule.java

```diff
@@ -41,9 +42,9 @@ public IntervalLoadRule(
       @JsonProperty("tieredReplicants") Map<String, Integer> tieredReplicants
   )
   {
-    validateTieredReplicants(tieredReplicants);
+    this.tieredReplicants = tieredReplicants == null ? ImmutableMap.<String, Integer>of() : tieredReplicants;
```

Comment:
More specifically ImmutableMap.of(DruidServer.DEFAULT_TIER, DruidServer.DEFAULT_NUM_REPLICANTS)